src/parallel.jl

```diff
@@ -70,7 +70,7 @@ end
 # since  `ThreadedEx` has shown to be more performant. This may
 # change in the future.
 # See PR 33 https://github.com/JuliaML/MLUtils.jl/pull/33
-_default_executor() = ThreadedEx(basesize=1)
+_default_executor() = TaskPoolEx(basesize=1, background=true)
```

Comment:
To get something that runs even on single-threaded Julia setups:

```suggestion
_default_executor() = TaskPoolEx(basesize=1, background=Threads.nthreads() > 1)
```